examples/workflow/1_create_model.ipynb

```diff
@@ -20,21 +20,25 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Your analysis starts by defining an amplitude module that describe the \n",
-    "intermediate resonances in the decay process you study. For this, of course,\n",
-    "you first have to determine what resonances there could be.\n",
+    "Your analysis starts by defining an amplitude model that describes the\n",
+    "reaction process you want to study. Such a model is generally very complex\n",
+    "and requires a fair amount of effort by the analyst (you). This also gives a\n",
+    "lot of rooms for mistakes.\n",
```

Comment:
*lot of room

Reply:
https://github.com/ComPWA/pycompwa/pull/74/commits/f0d0f8c4b64a5afcfb5aab152471f61e97030ee4